app/views/students/_student_form.html.haml

```diff
@@ -1,9 +1,12 @@
 %tr
   %td.pb-1.pt-3
     %p.lead= form_type.to_s.titleize
-  - if @student.valid_forms(form_type).any?
+  - if form_type == :medicals && @student.medicals.any?
```

Comment:
This condition does not tell you whether they have a valid medical

Reply:
I think this is resolved